src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
I guess we can defer that investigation for later, when we will decide to make it actualy extensible, good move in any case :)